_posts/2016-09-22-maturing-the-security-project.markdown

```diff
@@ -0,0 +1,162 @@
+new opportunities to innovate on security and create new ways to make OpenStack more secure.
+
+## Score Card
+I'm sure this isn't a new idea but I wasn't able to find much good evidence of
+other teams doing this when I had a quick search.
+
+I'm proposing a score card for the security project, to ensure we're doing all that
+we should be doing and identify those areas where we need to improve. I've based this
+on the [Project Team Guide](http://docs.openstack.org/project-team-guide)
+
+Requirement | Status | Notes |
+------------|--------|-------|
+Open Code | Achieved | All code in git and licensed appropriately
+Open Design | Achieved | All design is open to the public, conducted at summits etc
+Open Development | Achieved | We follow standard OpenStack best practice
+Open Community | Needs Improvement | We have a gap around the mailing lists that we need to address
+Public Meetings on IRC | Achieved | 1700UTC Thursdays #openstack-meeting-alt
+Project IRC channel | Achieved | #openstack-security
+Community Support Channels | Mostly Achieved | We are strong on Launchpad and IRC which is where 90% of our workload comes from however we need to pay more attention to the ML and ask.openstack.org
+Planet OpenStack | Achieved | This security blog posts to planet openstack
+Participate in Design Summits | Achieved | Regular, very well attended sessions
+Participate in Cross Project meeting | Needs Improvement | We did, for a time have people attending these meetings but not much was actively invested and over time I think they just fell off peoples calendars.
+Release Management | Achieved | We have a number of software projects that we created to support or enhance security in OpenStack. As they're not directly consumed by OpenStack Operators they've not been part of the normal release cycle. Instead we follow the Independent release model.
+Support Phases | Needs Clarification | I'm not sure if our software projects would fall within this as we have an independent release model.
+Testing | Achieved | All of our software and documentation efforts have appropriate gate tests in place. Functional and Unit tests are in place where appropriate.
+Vulnerability Management | Achieved | Our software projects don't have the vulnerability managed tag, however as the OSSP we do triage any security issues in our own software following standard processes, this is best demonstrated with the recent XSS issue in Bandit https://bugs.launchpad.net/bandit/+bug/1612988
+Documentation | Mostly Achieved | We have a lot of documentation out there for customers and consumers of openstack [OSSNs](https://wiki.openstack.org/wiki/Security_Notes), [security.openstack.org](https://security.openstack.org), the [security guide](http://docs.openstack.org/sec/) as well as developer documentation such as that for [Anchor](http://docs.openstack.org/developer/anchor/) and [Bandit](http://docs.openstack.org/developer/bandit/)
```

Comment:
In the previous items, you've pointed out what is needed for something to be achieved (if it wasn't achieved). What's missing here? Why haven't we achieved this?